Sources/Penny/BotStateManager.swift

```diff
@@ -68,5 +127,23 @@ actor BotStateManager {
     func _tests_setDisableDuration(to duration: Duration) {
         self.disableDuration = duration
     }
+
+    func _tests_didShutdownSignalEventContent() -> String {
+        makeSignalMessage(text: StateManagerSignal.didShutdown.value, id: self.id - 10)
+    }
 #endif
 }
+
+enum StateManagerSignal {
+    case shutdown
+    case didShutdown
+
+    var value: String {
+        switch self {
+        case .shutdown:
+            return "Hello the other Pennys 👋 you can retire now :)"
+        case .didShutdown:
+            return "I'm retired!"
+        }
+    }
+}
```

Comment:
Whilst I appreciate the personality of these messages (and you know I do! 🤣), perhaps this?
```suggestion
enum StateManagerSignal: String, CustomStringConvertible {
    case shutdown
    case didShutdown

    /// See ``Swift/CustomStringConvertible/description``.
    var description: String {
        switch self {
        case .shutdown:
            return "Hello the other Pennys 👋 you can retire now :)"
        case .didShutdown:
            return "I'm retired!"
        }
    }
}
```
(then use `.rawValue` where you were using `.value` elsewhere)